app/controllers/users/registrations_controller.rb

```diff
@@ -16,6 +16,11 @@ def create
 
     if resource.save
       if resource.active_for_authentication?
+        last_ip = request.env['REMOTE_ADDR']
```

Comment:
for logging I check in a different order which I found at the time to yield better results than checking REMOTE_ADDR first: `HTTP_X_FORWARDED_FOR, HTTP_X_CLUSTER_CLIENT_IP, REMOTE_ADDR`